tests/test_schema_functions.py

```diff
@@ -310,7 +310,13 @@ def test_schema_functions(function, args_kwargs, expected_schema):
         pass
 
     # also build the serializer, just to check it doesn't raise an error
-    SchemaSerializer(schema)
+    if schema['type'] not in {
+        'validator-function-before',
+        'validator-function-after',
+        'validator-function-wrap',
+        'validator-function-plain',
+    }:
```

Comment:
This exclusion is because these types are no longer supported in serialization, because of the split?

Do we need a similar test for serialization schemas?